packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -41,12 +51,22 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
 
                 if (currency === network.symbol) {
                     // csv amount in crypto currency
-                    output.amount = item.amount || '';
+                    const cryptoValue = item.amount || '';
+                    if (!isInteger(cryptoValue) && shouldSendInSats) {
```

Comment:
if you are using sats (shoudSendInSats) and value **is not** an integer (float) then value should be transformed to sats

Reply:
Ok I get it. The condition could be improved, possibly checking for dust limit instead of integer, and doing the conversion for either all or none of the outputs. But super low priority, you can leave it as it is...

Trying to send 1 BTC:

https://github.com/trezor/trezor-suite/assets/42465546/2acc6732-6329-4df9-bfd8-ae110a7990c5

Reply:
not sure how do you want to improve it.

if suite is set to **using sats** and data file says `amount=1` how suite should know that in fact  you are requesting 1 btc (100000000 sats) ? or another example if data file says `amount=746` how suite should decide that you are requesting 746 sats or 746 btc?

this condition covers **only the opposite case**: if amount **is not** an integer (let say 1.1 btc) then automatically converts it into sats otherwise assume that this integer represents sats

Reply:
If it used dust limit instead of `isInteger` in the condition, then `1` would be interpreted as BTC - Suite does not let you send 1 sat anyway. `746`, on the other hand, is above dust limit, so this would be interpreted as sats - one does not usually send more than 500 BTC, at least I don't do it often.

But it is probably not worth implementing, importing a transaction of 1 BTC is not likely either.

Reply:
well, you are right. 
since its confusing why decimal amounts are converted and integer amounts are not im removing this condition (tbh i dont exactly remember why i did it like this) 
now **all amounts** will be converted to satoshis :+1: 

[8ff322f](https://github.com/trezor/trezor-suite/pull/10327/commits/8ff322fbc9f21870cfa51485397fdcd68c9f10c3)

Reply:
Ok but this means that the amount in the CSV must always be in BTC and we don't state it anywhere I think 😬

Reply:
we do state it in `Show CSV example`:

```
address,amount,currency
tb1qp4v7ajc36ehssrqq2thhslp4apxegsp030f9kt,0.31337,BTC
tb1qqdmj0nuavjmr5f7huu8n9rh6wke4c2r4ykrj50,0.1,USD
```

you can either use BTC or FIAT currency, in fact there is no mention that you can use SAT currency in csv - which will be new feature

issue #7263 is about missing calculation from imported BTC to SAT after the import

Reply:
Ok, let's merge it.